sensu-plugins-ipmi.gemspec

```
@@ -27,7 +27,7 @@ Gem::Specification.new do |s|
   s.test_files             = s.files.grep(%r{^(test|spec|features)/})
   s.version                = SensuPluginsIpmi::Version::VER_STRING
 
-  s.add_runtime_dependency 'sensu-plugin',  '~> 1.2'
+  s.add_runtime_dependency 'sensu-plugin',  '>= 1.2'
```

As there are a number of breaking changes in 1.x, 2.x, 3.x, and 4.x I think its better if we bump to 4.x so that users know what behavior to expect if multiple versions of `sensu-plugin` is installed.
```suggestion
  s.add_runtime_dependency 'sensu-plugin',  '~> 4.0'
```

---

My issue has been that a number of other sensu plugins don't support 4.0 so it will require a bit of work on this side to allow them to run in their own environments. This particular change shouldn't have made it up as part of this PR - apologies. I'll fix it.

---

It should not matter if all the plugins don't support `~> 4.0` yet, while each of those major versions contains a breaking change they are all possible to run in the same environment. It's pretty common to install multiple versions of `sensu-plugin` on your server and let each plugins gemspec define what it needs, when it runs its runtime constraints will select the proper version.

---

It's shouldn't matter, but because we're clumping all the plugins in via a single Gemfile, we get conflicts. It's simply due to how we are currently packaging things here which we will need to change.